app/views/books/_form.html.slim

```diff
@@ -0,0 +1,51 @@
+= simple_form_for @book do |f|
+
+  = f.input :title, label: 'Title',
+                   placeholder: 'Title',
+                   input_html: { class: 'form-control' },
+                   wrapper_html: { class: 'form-group.book-form' }
+
+  = f.input :publishing_house, label: 'Publishing house',
+                               placeholder: 'Publishing house',
+                               input_html: { class: 'form-control' },
+                               wrapper_html: { class: 'form-group.book-form' }
+
+  = f.input :language, label: 'Language',
+                       placeholder: 'Language',
+                       input_html: { class: 'form-control' },
+                       wrapper_html: { class: 'form-group.book-form' }
+
+  = f.input :year_of_publication, label: 'Year of publication',
+                                  placeholder: '****',
```

Comment:
probably, you are looking for input masks
that is not a placeholder responsibility